package.json

```
@@ -13,13 +13,13 @@
     "lint:wc": "polymer lint -i \"components/**/*.js\" \"demo/**/*.js\" \"test/**/*.js\"",
     "langs:rename": "mfv rename",
     "langs:sync": "mfv add-missing && mfv remove-extraneous",
-    "test:karma": "karma start",
-    "test:karma:watch": "karma start --auto-watch=true --single-run=false",
-    "test:polymer:local": "polymer test -p --skip-plugin sauce",
+    "test:headless": "web-test-runner --group default",
+    "test:headless:watch": "web-test-runner --group default --config web-test-runner.config.mjs",
+    "test:polymer:local": "polymer test -p --skip-plugin sauce  --config web-test-runner.config.mjs",
```

I think you want something like this:

```suggestion
    "test:headless": "web-test-runner --config web-test-runner.config.mjs",
    "test:headless:watch": "web-test-runner --config web-test-runner.config.mjs --watch",
    "test:polymer:local": "polymer test -p --skip-plugin sauce",
```

`polymer test` isn't going to understand the `@web/test-runner` config file -- it uses the older `wct.conf.json` and I think we just want to leave it alone unless we plan to rewrite all those test HTML files to the new format. Here's an [example of doing that](https://github.com/BrightspaceUI/navigation/pull/127) and it's a fair amount of work.